src/Symfony/Component/Form/FormRegistry.php

```diff
@@ -26,24 +26,16 @@ class FormRegistry implements FormRegistryInterface
     /**
      * @var FormExtensionInterface[]
      */
-    private $extensions = [];
+    private array $extensions = [];
 
     /**
      * @var ResolvedFormTypeInterface[]
      */
-    private $types = [];
+    private array $types = [];
 
-    /**
-     * @var FormTypeGuesserInterface|false|null
-     */
-    private $guesser = false;
-
-    /**
-     * @var ResolvedFormTypeFactoryInterface
-     */
-    private $resolvedTypeFactory;
-
-    private $checkedTypes = [];
+    private FormTypeGuesserInterface|null|false $guesser = false;
```

Comment:
false seems to be used as lazy loading value. we should use uninitialized property instead like in other places.

Reply:
the issue is that `isset($this->guesser)` also returns `false` for `null`

Reply:
I see. Really does not seem possible then without using reflection. PHP is kinda missing an is_unitialized function.

Reply:
Correct. The problem is that `null` is a valid value for `$guesser`. And unfortunately all operators that are safe on uninitialized properties (`isset()`, `??`, `??=`) behave the same on initialized properties that are set to `null`. I would need to use reflection or catch exceptions to check if the property is initialized.